source/versionInfo.py

```diff
@@ -5,43 +5,9 @@
 #See the file COPYING for more details.
 

```

Comment:
Perhaps a module level doc string to say "This module is for UI specific / translatable version information. The source of this information should come from the non-translated buildVersion module so that it can be used in parts of nvda that are executed pre-translation system initialisation."